kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -22,18 +22,20 @@
 
     <span class="visuallyhidden" v-if="subtopics.length">{{ $tr('navigate') }}</span>
 
-    <card-grid v-if="contents.length">
-      <content-grid-item
-        v-for="content in contents"
-        v-show="selectedFilter.value === 'all' || selectedFilter.value === content.kind"
-        :key="content.id"
-        class="card"
-        :title="content.title"
-        :thumbnail="content.thumbnail"
-        :kind="content.kind"
-        :progress="content.progress"
-        :link="genLink(content)"/>
-    </card-grid>
+    <content-card-grid :contents="contents" v-if="contents.length">
+
+      <template scope="content">
```

Comment:
the slot is optional. I put it in to avoid a refactor of the `search` and `explore` page that @rtibbles and @christianmemije worked on. They're using `v-show` rather than `v-if`, so the array they're passing in would have to change.